core/common/src/main/java/alluxio/network/protocol/databuffer/DataByteArrayChannel.java

```diff
@@ -52,6 +52,16 @@ public ByteBuffer getReadOnlyByteBuffer() {
   }
 
   @Override
+  public void readBytes(byte[] dst, int dstIndex, int length) {
+    throw new RuntimeException("DataByteArrayChannel#readBytes is not implemented.");
```

Comment:
` UnsupportedOperationException` please, same comment for all other `RuntimeException` in this PR.